src/nat/front_ends/fastapi/routes/hitl.py

```
@@ -0,0 +1,185 @@
+        async with self._lock:
+            self._pending_by_session[session_id].append(prompt)
+            fut: asyncio.Future[HumanResponse] = asyncio.get_event_loop().create_future()
+            self._response_waiters[prompt.id] = fut
+            await self._notify_subscribers(session_id, prompt)
+            return fut
+
+    async def respond(self, interaction_id: str, response: HumanResponse) -> None:
+        async with self._lock:
+            fut = self._response_waiters.get(interaction_id)
+            if fut is None:
+                raise KeyError(f"Interaction {interaction_id} not found")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)